nvflare/apis/workspace.py

```diff
@@ -18,34 +18,110 @@
 
 
 class Workspace:
-    def __init__(self, root_dir: str, name: str, config_folder: str):
+    def __init__(self, root_dir: str, site_name: str = "", config_folder: str = "config"):
```

Comment:
can site_name be `""` in our current code?

We should probably remove this later I agree.
(deployed app just call "app" instead of "app_server")

Reply:
future enhancement, we can discuss.

Reply:
can site_name be "" in our current code? if it can't we should remove this default value

Reply:
There are quite several places which are not providing the "site_name" when migrating the historical "workspace" folder into the Workspace object. Those cases don't require the "site_name". For the time being of testing authz function, leave like this for future enhancement.